vg/vgimg/vgimg.go

```diff
@@ -146,6 +146,8 @@ func UseDPI(dpi int) option {
 // the canvas from. The
 // minimum point of the given image
 // should probably be 0,0.
+//
+// Note that a copy of the input image is performed.
```

Comment:
I'd add that this means that writes to the canvas are not reflected in the original image.